client/cypress/end-to-end/ballot-comparison-failure-cases.spec.js

```diff
@@ -0,0 +1,238 @@
+    cy.findByText('Replace File').click()
+    cy.findAllByText('Upload File').should('have.length',2)
+    cy.fixture('CSVs/manifest/ballot_comparison_manifest.csv').then(fileContent => {
+      cy.get('input[type="file"]').first().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_manifest.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').spread((firstButton, secondButton) => {
+      firstButton.click()
+    })
+    cy.contains("Upload successfully completed")
+
+    // upload invalid cvr
+    cy.fixture('CSVs/cvr/ballot_comparison_cvr_col_error.csv').then(fileContent => {
+      cy.get('input[type="file"]').last().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_cvr_col_error.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').last().click()
+    cy.contains(/Invalid contest name/)
+    
+    // now upload valid cvr
+    cy.findAllByText('Replace File').spread((firstButton, secondButton) => {
+      secondButton.click()
+    })
+    cy.findByText('Upload File')
+    cy.fixture('CSVs/cvr/ballot_comparison_cvr.csv').then(fileContent => {
+      cy.get('input[type="file"]').last().attachFile({
+        fileContent: fileContent.toString(),
+        fileName: 'ballot_comparison_cvr.csv',
+        mimeType: 'csv'
+      })
+    })
+    cy.findAllByText('Upload File').click()
+    cy.findAllByText(/Upload successfully completed/).should('have.length', 2)
+
+    cy.logout(jurisdictionAdmin)
+    cy.loginAuditAdmin(auditAdmin)
+    cy.findByText(`TestAudit${id}`).click()
+    cy.findByText('Review & Launch').click()
+    cy.findByText('Launch Audit').click()
+    cy.findAllByText('Launch Audit').spread((firstButton, secondButton) => {
+      secondButton.click()
+    })
+    cy.findByRole('heading', {name: "Audit Progress"})
+    cy.logout(auditAdmin)
+    cy.loginJurisdictionAdmin(jurisdictionAdmin)
+    cy.findByText(`Jurisdictions - TestAudit${id}`).siblings('button').click()
+    cy.contains('Number of Audit Boards')
+    cy.findByText('Save & Next').click()
+    cy.findByText('Download Audit Board Credentials').click()
+    cy.logout(jurisdictionAdmin)
+    cy.task('getPdfContent', `cypress/downloads/Audit Board Credentials\ -\ Death Star\ -\ TestAudit${id}.pdf`).then((content) => {
+    function urlify(text) {
+      var urlRegex = /(((https?:\/\/)|(www\.))[^\s]+)/g;
+      return text.match(urlRegex, function(url) {
+      return url
+      }) 
+    }
+    board_credentials_url = urlify(content.text);
+    cy.visit(board_credentials_url[0])
+    cy.findAllByText('Audit Board Member').eq(0).siblings('input').type('Board Member 1')
+    cy.findAllByText('Audit Board Member').eq(1).siblings('input').type('Board Member 2')
+    cy.findByText('Next').click()
+    cy.contains(/Ballot Cards to Audit/)
+    cy.findByText('Start Auditing').click()
+
+    // submit empty ballot review
```

Comment:
similar comment here - should you even be able to go to the review screen without providing an interpretation?

Reply:
Good point!